paper/utils.md

```diff
@@ -0,0 +1,64 @@
+``` r
+library(funkyheatmap)
+library(tidyverse)
+```
+
+    ## ── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
+    ## ✔ dplyr     1.1.4     ✔ readr     2.1.5
+    ## ✔ forcats   1.0.0     ✔ stringr   1.5.1
+    ## ✔ ggplot2   3.5.1     ✔ tibble    3.2.1
+    ## ✔ lubridate 1.9.4     ✔ tidyr     1.3.1
+    ## ✔ purrr     1.0.2     
+    ## ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
+    ## ✖ dplyr::filter() masks stats::filter()
+    ## ✖ dplyr::lag()    masks stats::lag()
+    ## ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
+
+``` r
+legends <- list(
+  list(title = "Bar", palette = "qc", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "benchmark", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "scaling", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "stability", enabled = FALSE, geom = "bar"),
+  list(title = "Error reason", palette = "error_reasons", enabled = TRUE, geom = "pie"),
```

Comment:
I forget -- do we have knobs to make this geom wider?

Reply:
Not for the bar geom legend, and I presume not for any legend? It could be an interesting addition.

Reply:
Sorry, I didn't mean to highlight the bar geoms.

What I noticed is that the pie legend is cut off:

![Screenshot from 2024-12-13 16-27-24](https://github.com/user-attachments/assets/a95f00ee-5f7b-47aa-90c1-483f5b5260b2)

Reply:
Ah right, yeah we do have a setting for that ^^